HWReleaseProcess.md

```diff
@@ -0,0 +1,72 @@
+# HW Release process
+
+## Nightly
+
+1. When the **HW Release Workflow** is triggered, the **RTL Submodule Pointer Update** flow is called
+2. The **nightly-verilator** smoke test regression test will be executed against the HEAD of the **main** branch
+    -  If the the test passes a **release collateral** archive will be generated and pushed as a Github Artifact
+    -  Repo tag will be added as well in the form "release_v#yyyy#mm#dd_#n"
+
+## RTL Submodule Pointer Update
+
+1. The RTL submodule pointer is compared against the **HEAD of main** of the **caliptra RTL** repo
+    - In case they both point to the same **SHA commit** the rest of the flow is skipped 
+2. A new commit will update the RTL submodule pointer to point to the HEAD of the main branch, and update the registers RDL
+3. A new **nightly release** branch will be created from the HEAD of the **main** branch in the **caliptra-sw** repo
+   - Naming of the branch will indicate it is a nightly release and will have a target point release date (ci_rtl_#yyyy#mm#dd_#n)
+4. The branch wil trigger all the CI tests required to submit a PR
+5. Upon succesfully completion of the CI tests, the branch will be merged and deleted
+
+## Major Releases
+
+1. The **release collateral** will be manually handed over the RTL team to be verified in the DV environment
```

Comment:
Does the "DV environment" also test that the RTL can be synthesized properly by the various synthesis tools out there?

Reply:
The "DV environment" should check that the RTL can be synthesized with the tool chosen by the WG. As Caliptra being an open source project, we should/could expect an integrator to find a problem with a different tool to file an issue, and maybe contribute ?

Reply:
Looks good.